src/picongpu/include/particles/ionization/byField/BSI/BSI_Impl.hpp

```diff
@@ -118,7 +118,7 @@ namespace ionization
              * during loop execution. The reason for this is the `__syncthreads()` call which is necessary after
              * initializing the E-/B-field shared boxes in shared memory.
              */
-            DINLINE void init(const DataSpace<simDim>& blockCell, const int& linearThreadIdx, const DataSpace<simDim>& totalCellOffset)
+            DINLINE void init(const DataSpace<simDim>& blockCell, const int& linearThreadIdx, const DataSpace<simDim>& localCellOffset)
```

Comment:
Please add a short description to your params that everboy knows what `blockCell`,... is.

```
@param blockCell Where is the origin?
@param linearThreadIdx thread id inside the block
@param localCellOffset  Where is the origin? I think the origin is the BORDER
```